2200075/4_10505 . 소득 불균형.py

```diff
@@ -0,0 +1,12 @@
+# 4. 
+T = int(input())
+
+for t in range(1,T + 1) :
+    N = int(input())
+    cnt = 0
+    numbers = list(map(int,input().split()))
+    for number in numbers :
+        if sum(numbers) / len(numbers) >= number :
+            cnt +=1
```

Comment:
깔끔한 풀이네요! 👍 
```suggestion
    avg = sum(numbers) / len(numbers)
    for number in numbers:
        if avg >= number:
            cnt += 1
```
평균을 `for` 문 밖에서 한 번만 계산하는 방법도 있습니다!

Reply:
가독성이 확 좋아졌네요 감사합니다